src/com/jetbrains/SystemUtils.java

```diff
@@ -28,4 +28,14 @@ public interface SystemUtils {
      * Implemented for G1 only; equivalent to System.gc() for other garbage collectors.
      */
     void fullGC();
+
+    /**
+     * Performs garbage collection making an additional effort to shrink the heap in size.
+     * After GC the heap size is set proportional to heap used bytes using
```

Comment:
What do you mean by `the heap size is set`? There are two heap sizes: minimum and maximum. There's also the current used amount, but I'm sure it cannot be regulated by an option as it is defined by the running application itself.

Reply:
I clarified the doc:

```
Performs garbage collection making an additional effort to reduce the heap's committed size.
After GC, the JVM adjusts the committed heap size based on the amount of used memory, adhering to the -XX:JbrShrinkingGcMaxHeapFreeRatio option.
...
Note: This does not change the minimum (-Xms) or maximum (-Xmx) heap size limits
```

It it more clear now?

Reply:
Yes, thank you!